tests/ui/UnreadIndicatorsTest.js

```diff
@@ -259,9 +263,15 @@ describe('Unread Indicators', () => {
             .then(() => {
                 // Verify the LHN is now open
                 expect(areYouOnChatListScreen()).toBe(true);
+
                 // Tap on the chat again
                 return navigateToSidebarOption(0);
             })
+            .then(() => {
+                // Sending event to clear the unread indicator cache, given that the test doesn't behave as the app
```

Comment:
> given that the test doesn't behave as the app
What does this mean?

Reply:
When there is a single panel, in the tests, seems that it keeps all components in memory, so I need to reset the state to make it work, "simulating" the same as what the app does